ginepro/Cargo.toml

```
@@ -15,7 +15,7 @@ async-trait = "0.1"
 http = "0.2"
 thiserror = "1"
 tokio = { version = "1", features = ["full"] }
-tonic = { version = "0.10", features = ["tls"] }
+tonic = { version = "0.11", features = ["tls"] }
```

Can we use a range dependency here to avoid having to make this a breaking change?

```suggestion
tonic = { version = ">=0.10,<0.12", features = ["tls"] }
```